backend/init_data/skills/browser/_base.py

```
@@ -0,0 +1,375 @@
+        This should be called when the task is completed to clean up resources.
+
+        Args:
+            task_id: Task ID to remove
+        """
+        if task_id in cls._instances:
+            logger.info(
+                f"[BrowserSessionManager] Removing instance for task_id={task_id}"
+            )
+            instance = cls._instances[task_id]
+            # Cleanup is async, but we'll schedule it
+            asyncio.create_task(instance.close())
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)